hw/ip/edn/doc/_index.md

```diff
@@ -203,13 +203,12 @@ The following waveform shows an example of how the peripheral hardware interface
 {signal: [
    {name: 'clk'           , wave: 'p...|.........|.......'},
    {name: 'edn_enable'    , wave: '01..|.........|.......'},
-   {name: 'edn_req'       , wave: '0..1|......0.1|.....0.'},
-   {name: 'edn_ack'       , wave: '0...|.1.01.0..|101....'},
-   {name: 'edn_bus[31:0]' , wave: 'x...|.33.33...|3.3333.', data: ['gb0','gb1','gb2','gb3','es4','es5','es6','es7','es8']},
+   {name: 'edn_req'       , wave: '0..1|..01.0..1|.....0.'},
+   {name: 'edn_ack'       , wave: '0...|.10.10...|....10.'},
+   {name: 'edn_bus[31:0]' , wave: 'x...|.3x.3x...|....3x.', data: ['es0','es1','es2']},
```

Comment:
These should not be "x"'s.  You may not care what the value is, but some IP consuming them might.

Reply:
Assuming exact details matter for security integrity review, the waveform for request/ack has been updated with implementation specifics - x's have been removed.